spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-ecommerce-tracker.spec.js

```diff
@@ -321,6 +322,14 @@ describe('Google Analytics 4 ecommerce tracking', function () {
       resultToBeClicked.click()
       expect(window.dataLayer[3].event).toBe(onSearchResultClickExpected.event)
       expect(window.dataLayer[3].search_results).toEqual(onSearchResultClickExpected.search_results)
+
+      var differentListItemName = 'different list item name'
```

Comment:
I think it might be worth splitting this out into a separate test, to highlight what this functionality is.

Reply:
That makes sense, have made this change!